SECRETS.md

```diff
@@ -1,5 +1,11 @@
 ## Managing Secrets
 
+Secrets come in two flavors, image-pull secrets and application secrets.  Kubernetes uses [image-pull secrets to facilitate pulling docker images from a private registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).  Application secrets are simply sensitive values that your function requires to function, e.g. a database password. `faas-netes` supports both types of secrets.  If you are familiar with the inner workings of Kubernetes, then image-pull secrets will be used as you would expect from the Kubernetes documentation. Generic application secrets will behave in the same way that secrets are handled in Docker Swarm, specifically, they will be mounted as files in the `/run/secrets` folder.  This allows your functions to be agnostic of the deployment infrastructure. The relevant Kubernetes documentation for how this is implemented can be found [here](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod).
```

Comment:
Capitalise docker -> Docker

Prefer plain English:

> Kubernetes uses [image-pull secrets to facilitate pulling docker images from a private registry

> Kubernetes uses [image-pull secrets to pull images from private registries]

Can this be broken up onto multiple lines?

Reply:
Just pushed a change that should clean this up.